test/snapshots/test.js.md

```diff
@@ -36,6 +37,187 @@ Generated by [AVA](https://avajs.dev).
       _: [],
     }
 
+## json
+
+> Snapshot 1
+
+    {
+      assets: [
+        {
+          chunkNames: [
+            'main',
+          ],
+          chunks: [
+            'main',
+          ],
+          emitted: true,
+          info: {},
+          name: 'main.js',
+          size: 3875,
+        },
+      ],
+      assetsByChunkName: {
+        main: 'main.js',
+      },
+      children: [],
+      chunks: [
+        {
+          children: [],
+          childrenByOrder: {},
+          entry: true,
+          files: [
+            'main.js',
+          ],
+          filteredModules: 0,
+          hash: 'ee8ce49d6862106d56fe',
+          id: 'main',
+          initial: true,
+          modules: [
+            {
+              assets: [],
+              built: true,
+              cacheable: true,
+              chunks: [
+                'main',
+              ],
+              depth: 0,
+              errors: 0,
+              failed: false,
+              id: './entry.js',
+              identifier: '/Users/juhovepsalainen/Projects/github/webpack-nano/test/fixtures/entry.js',
```

Comment:
It looks like these should go as well as it's not good to have them in the snapshot.